src/Microsoft.VisualBasic/tests/ConversionsTests.cs

```diff
@@ -5,83 +5,2952 @@
+
+            // ushort.
+            yield return new object[] { ushort.MinValue, ushort.MinValue };
+            yield return new object[] { (ushort)1, (ushort)1 };
+            yield return new object[] { ushort.MaxValue, ushort.MaxValue };
+            yield return new object[] { (UShortEnum)ushort.MinValue, ushort.MinValue };
+            yield return new object[] { (UShortEnum)1, (ushort)1 };
+            yield return new object[] { (UShortEnum)ushort.MaxValue, ushort.MaxValue };
+
+            // short.
+            yield return new object[] { (short)0, ushort.MinValue };
+            yield return new object[] { (short)1, (ushort)1 };
+            yield return new object[] { short.MaxValue, (ushort)32767 };
+            yield return new object[] { (ShortEnum)0, ushort.MinValue };
+            yield return new object[] { (ShortEnum)1, (ushort)1 };
+            yield return new object[] { (ShortEnum)short.MaxValue, (ushort)32767 };
+
+            // uint.
+            yield return new object[] { uint.MinValue, ushort.MinValue };
+            yield return new object[] { (uint)1, (ushort)1 };
+            yield return new object[] { (UIntEnum)uint.MinValue, ushort.MinValue };
+            yield return new object[] { (UIntEnum)1, (ushort)1 };
+
+            // int.
+            yield return new object[] { 0, ushort.MinValue };
+            yield return new object[] { 1, (ushort)1 };
+            yield return new object[] { (IntEnum)0, ushort.MinValue };
+            yield return new object[] { (IntEnum)1, (ushort)1 };
+
+            // ulong.
+            yield return new object[] { ulong.MinValue, ushort.MinValue };
+            yield return new object[] { (ulong)1, (ushort)1 };
+            yield return new object[] { (ULongEnum)ulong.MinValue, ushort.MinValue };
+            yield return new object[] { (ULongEnum)1, (ushort)1 };
+
+            // long.
+            yield return new object[] { (long)0, ushort.MinValue };
+            yield return new object[] { (long)1, (ushort)1 };
+            yield return new object[] { (LongEnum)0, ushort.MinValue };
+            yield return new object[] { (LongEnum)1, (ushort)1 };
+
+            // float.
+            yield return new object[] { (float)0, ushort.MinValue };
+            yield return new object[] { (float)1, (ushort)1 };
+
+            // double.
+            yield return new object[] { (double)0, ushort.MinValue };
+            yield return new object[] { (double)1, (ushort)1 };
+
+            // decimal.
+            yield return new object[] { (decimal)0, ushort.MinValue };
+            yield return new object[] { (decimal)1, (ushort)1 };
+
+            // bool.
+            yield return new object[] { true, ushort.MaxValue };
+            yield return new object[] { false, ushort.MinValue };
+
+            // null.
+            yield return new object[] { null, ushort.MinValue };
+
+            // string.
+            yield return new object[] { "0", ushort.MinValue };
```

Comment:
same thing. please do the same thing everywhere

Reply:
Fixed